sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -58,6 +58,11 @@ struct<plan:string>
 
 (1) Scan parquet default.explain_temp1 
 Output: [key#x, val#x]
+Batched: true
+Format: Parquet
```

Comment:
This is already in the node name `Scan parquet default.explain_temp1`

Reply:
@cloud-fan I can filter it. But wanted to double check first. The one we print in the node name is `relation.toString` and the one printed here is `relation.format.toString`. Are they going to be same always ?

Reply:
`relation.toString` always contain the format name AFAIK.

Reply:
@cloud-fan OK.. sounds good wenchen. I will drop this then.